### PR TITLE
404 page

### PR DESCRIPTION
**Pull request by shamilkotta**

### CODE REVIEW

File: src/layouts/GuestLayout.jsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { Route, Switch } from "react-router";
+import { Redirect, Route, Switch } from "react-router";
```
Comment (Muhammed-Rahif) [inline src/layouts/GuestLayout.jsx:2]:
```suggestion
import { Redirect, Route, Switch } from "react-router-dom";
```

**Review by Muhammed-Rahif — APPROVED** [review]

**Comment by Muhammed-Rahif** [comment]
@shamilkotta 404 page must be in `App.js`, because it must be out of layout... Also want use for developer and buyer cases..

**Review by Muhammed-Rahif — APPROVED** [review]
lgtm

**Review by ajeeshcp — APPROVED** [review]
The page looks good

**Comment by Muhammed-Rahif** [comment]
Merging!

**Pull request merged**